tools/release-tools/setVersion.js

```diff
@@ -321,7 +321,7 @@ function updateVersion(src, orgVersion, newVersion) {
 }
 
 const setPackageJsonRelease = () => {
-    const files = globby.sync("./**/package.json");
+    const files = globby.sync(["./**/package.json", "!**/node_modules/**"]);
```

Comment:
Performance improvement for automatic  version update script, tells it to ignore node_modules which makes it a LOT faster.